cmd/mdmctl/apply.go

```diff
@@ -56,6 +60,7 @@ Apply a resource.
 Valid resource types:
 
   * blueprints
+  * dep-tokens
```

Comment:
Was get/apply only supposed to be blueprints? I thought it was a general interface for "getting" and "applying" (putting) resources into the MDM server (regardless of resource type).

Yeah I could see this, not opposed. Might adjust it slightly:

```
mdmctl dep token -list # list an abreviated list of our tokens
mdmctl dep token -import /path/to/token.p7m # import a token encrypted to our private key
mdmcel dep token -import-json /path/to/token.json # import a token, JSON
mdmctl dep token -export-json /path/to/exported.json # export a token, JSON
mdmctl dep token -export-json /path/to/t.json -ck CK_534323234... # export this particular token
mdmctl dep token -export-cert /path/to/cert.pem # export DEP token public key
```

But totally flexible here.

One thing I want to make sure of, though, is that the DEP tokens are plural. Any given MDM server ought to be able to support multiple DEP tokens. Perhaps things can appear singular to an end-user, but the backend ought to be able to support (or be designed in a way to support) multiple DEP tokens.